exporters/common/src/main/java/io/opentelemetry/exporter/internal/http/HttpExporterBuilder.java

```diff
@@ -124,6 +135,16 @@ public HttpExporterBuilder<T> setMeterProvider(Supplier<MeterProvider> meterProv
     return this;
   }
 
+  public HttpExporterBuilder<T> setHealthMetricLevel(HealthMetricLevel healthMetricLevel) {
+    this.healthMetricLevel = healthMetricLevel;
+    return this;
+  }
+
+  public HttpExporterBuilder<T> setComponentType(String componentType) {
```

Comment:
This is only used in one place to support `exportAsJson`. I think we should consider disabling internal telemetry in that case rather than jump through hoops to support it. Ostensibly, we don't support OTLP http/json export at all.